paywall/src/unlockTypes.ts

```diff
@@ -78,6 +78,7 @@ export interface MetadataInput {
   name: string
   type: 'text' | 'date' | 'color' | 'email' | 'url'
   required: boolean
+  protected?: true // optional, all non-protected fields are treated as public
```

Comment:
Since this is user centric data (from the owner) let;s play it safe and make it all protected for now.